styles/header/bottom-header.css

```
@@ -0,0 +1,39 @@
+.bottom-header {
+  display: flex;
+  height: 54px;
+  padding: 0px 98px 0px 101px;
+  justify-content: center;
+  align-items: center;
+  align-self: stretch;
```

제 생각에는 
```suggestion
  display: flex;
  align-items: center;
  height: 54px;
  padding: 0px 101px;
```

이렇게만 해도 되지않나요?

저렇게 코드를 짠 이유가 있을까요?

---

이유는 없습니당.. 피그마를 많이 참고해서.. 생각해보니 필요없을 거 같네요!! 수정하였습니당